packages/shortest/src/cli/bin.ts

```diff
@@ -128,10 +129,13 @@ async function main() {
   const targetUrl = args
     .find((arg) => arg.startsWith("--target="))
     ?.split("=")[1];
-  const testPattern = args.find((arg) => !arg.startsWith("--"));
+  const cliTestPattern = args.find((arg) => !arg.startsWith("--"));
   const debugAI = args.includes("--debug-ai");
 
   try {
+    const config = getConfig();
```

Comment:
seems like this code is giving the config error. you're calling `getConfig` before a call to initialize.